modules/ROOT/pages/resource-usage.adoc

```diff
@@ -0,0 +1,28 @@
+= View Runtime Fabric Status and Resources
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+You can use Anypoint Runtime Manager to view the resource usage and health of your Anypoint Runtime Fabric clusters.
```

Comment:
To avoid confusion, I'd suggest we remove "resource usage"

Reply:
Got it!